pkg/notify/mattermost.go

```diff
@@ -120,11 +112,25 @@ func (m *Mattermost) SendEvent(event events.Event) error {
 
 // SendMessage sends message to Mattermost channel
 func (m *Mattermost) SendMessage(msg string) error {
-	post := &model.Post{}
-	post.ChannelId = m.Channel
-	post.Message = msg
-	if _, resp := m.Client.CreatePost(post); resp.Error != nil {
-		log.Error("Failed to send message. Error: ", resp.Error)
+	// Set configurations for Mattermost server
+	client := model.NewAPIv4Client(m.MattermostUrl)
+	client.SetOAuthToken(m.MattermostToken)
+	botTeam, resp := client.GetTeamByName(m.MattermostTeam, "")
+	if resp.Error != nil {
+		return resp.Error
+	}
+	for _, channel := range m.getAllChannels() {
+		post := &model.Post{}
+		botChannel, resp := client.GetChannelByName(channel, botTeam.Id, "")
+		if resp.Error != nil {
+			log.Errorf("Unable to find the channel %v", channel)
+			return resp.Error
```

Comment:
Should `continue` here. We should at lease send notifications to the correct channels